e2e-tests/gef/cypress/e2e/facility/unissued-facilities/AIN/unissued-facilities-change-submit-ukef-AIN.spec.js

```diff
@@ -553,13 +552,9 @@ context('Submit to UKEF with unissued to issued facilities', () => {
       cy.url().should('eq', relative(`/gef/application-details/${dealId}#${unissuedFacilitiesArray[2]._id}`));
     });
 
-    it('should not contain already issued facility or submission message', () => {
+    it('should not contain already issued facilities', () => {
       applicationActivities.subNavigationBarActivities().click();
 
-      applicationActivities.activityTimeline().should('not.contain', PORTAL_ACTIVITY_LABEL.MIN_SUBMISSION);
-      applicationActivities.activityTimeline().should('not.contain', PORTAL_ACTIVITY_LABEL.MIA_SUBMISSION);
-      applicationActivities.activityTimeline().should('not.contain', PORTAL_ACTIVITY_LABEL.AIN_SUBMISSION);
-
```

Comment:
I've removed this as it's an odd thing to test: 
- If the database was populated correctly the activity timeline should contain MIN/MIA submission details 
- Checking that e.g. AIN details don't show on the activity timeline doesn't seem relevant to the test file.

@abhi-markan are you happy with me removing this?